src/react-elastic-carousel/components/Carousel.js

```diff
@@ -208,18 +208,21 @@ class Carousel extends React.Component {
     const {
       verticalMode,
       children,
-      itemsToShow
+      itemsToShow,
+      showEmptySlots
     } = this.getDerivedPropsFromBreakPoint();
     const { height: sliderHeight } = sliderNode.contentRect;
     const nextState = {};
     const childrenLength = Children.toArray(children).length;
     if (verticalMode) {
       const childHeight = sliderHeight / childrenLength;
-      // We use Math.min because don't want to make the child smaller
-      // if number of children is smaller than itemsToShow.
-      // Because we will have "empty slots"
+      // When "showEmptySlots" is false
+      // We use Math.min because we don't want to make the child smaller
+      // if the number of children is smaller than itemsToShow.
+      // (Because we do not want "empty slots")
       nextState.rootHeight =
-        childHeight * Math.min(childrenLength, itemsToShow);
+        childHeight *
+        (showEmptySlots ? itemsToShow : Math.min(childrenLength, itemsToShow));
```

Comment:
This will create an infinite loop when both `showEmptySlots` and  `verticalMode` are set to true and when `itemsToShow` is greater than `childrenLength`.

This happens because we based our `rootHeight` on `childHeight` and `childHeight` is based on `sliderHeight `, when the height changes, we trigger the resize observer listener again, which will make us increase the height, which will tigger the observer and so on...

I'm not sure whats the solution here yet, maybe this mechanics is wrong and need a revisit.
We don't have this problem when `verticalMode` is set to false because we do not set the `width` for the root, we only deal with the height. 
So maybe the solution is to tackle the height issue with CSS rather with us trying to figure out what `height` we need to set.

Reply:
It's strange, I tested it on the docs app with `showEmptySlots` and `verticalMode` set to true and it seemed to work... wasn't it the way it worked before https://github.com/sag1v/react-elastic-carousel/pull/105 was merged ?

Reply:
I just tried to recreate, when `showEmptySlots` is true, the behaviour the carousel had before #105 was merged :)

Reply:
So you see the issue?
Basically to reproduce the issue you also need the `itemsToShow` to be greater than the `childrenLength`.
I.e, if you got 6 items then `itemsToShow` should be 7+.

Reply:
I think I found it, yes... I didn't see it at first, but the height is indeed infinite... did this problem exist before #105 was merged ? I don't understand how it could've worked before, but now it doesn't :(

Reply:
And I don't want to break everything you did... maybe this prop should be incompatible with verticalMode too, juste like outerSpacing :/